sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -274,12 +274,18 @@ async Task OnBreakPointHit (JObject args, CancellationToken token)
 					var frames = new List<Frame> ();
 					int frame_id = 0;
 					var the_mono_frames = res.Value? ["result"]? ["value"]? ["frames"]?.Values<JObject> ();
+
 					foreach (var mono_frame in the_mono_frames) {
 						var il_pos = mono_frame ["il_pos"].Value<int> ();
 						var method_token = mono_frame ["method_token"].Value<int> ();
 						var assembly_name = mono_frame ["assembly_name"].Value<string> ();
 
 						var asm = store.GetAssemblyByName (assembly_name);
+						if (asm == null) {
+							Info ($"Unable to find assembly: {assembly_name}");
```

Comment:
If a stacktrace has many frames in the same missing assembly, we're going to dump this warning a bunch of times. Maybe not a big deal?

Reply:
we need to clean up the output a lot but this is the same as the missing method spew below it.

Reply:
Sounds totally fine then